patches/nginx-1.13.6-resolver_conf_parsing.patch

```diff
@@ -0,0 +1,201 @@
+diff --git a/src/core/ngx_resolver.c b/src/core/ngx_resolver.c
+index cd55520c..e61a6d53 100644
+--- a/src/core/ngx_resolver.c
++++ b/src/core/ngx_resolver.c
+@@ -5,6 +5,7 @@
+  */
+ 
+ 
++#include <resolv.h>
+ #include <ngx_config.h>
+ #include <ngx_core.h>
+ #include <ngx_event.h>
+@@ -15,6 +16,9 @@
+ #define NGX_RESOLVER_TCP_RSIZE  (2 + 65535)
+ #define NGX_RESOLVER_TCP_WSIZE  8192
+ 
++#define NGX_RESOLVER_FILE_BUF_SIZE  2048
```

Comment:
Note that there is not limitation for the resolv.conf file size. You could even writte poem in the comment! It would be better to put some comment here to mention that `2048` could cover most of cases.